packages/synapse-react-client/src/components/Forum/ForumPage.tsx

```diff
@@ -57,7 +57,7 @@ export const ForumPage: React.FC<ForumPageProps> = ({
   }
 
   return (
-    <div className="ForumTable bootstrap-4-backport">
+    <div className="ForumTable">
```

Comment:
Remove bootstrap class (inner table now uses `StyledTableContainer` instead of bootstrap)